pages/star-search/index.tsx

```diff
@@ -74,7 +74,7 @@ function ChatAvatar({ author, userId }: ChatAvatarProps) {
           alt="Your profile picture"
           width={32}
           height={32}
-          className="w-8 h-8 lg:w-10 lg:h-10 rounded-full"
+          className="w-8 h-8 rounded-full lg:w-10 lg:h-10"
```

Comment:
Not sure why there's all these changes. Do we have some plugin that auto arranges classes now?